src/app/shared/post-it-collections/post-it-collection.component/post-it-collection.component.ts

```diff
@@ -17,31 +17,33 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import {Component, EventEmitter, Input, OnDestroy, OnInit, Output} from '@angular/core';
+import {Component, EventEmitter, Input, OnChanges, OnDestroy, OnInit, Output, SimpleChanges, ViewChild} from '@angular/core';
 import {CollectionModel} from '../../../core/store/collections/collection.model';
-import {PostItLayout} from '../../utils/layout/post-it-layout';
 import {Workspace} from '../../../core/store/navigation/workspace.model';
 import {QueryConverter} from '../../../core/store/navigation/query.converter';
 import {QueryModel} from '../../../core/store/navigation/query.model';
 import {Role} from '../../../core/model/role';
 import {Subject, Subscription} from 'rxjs';
 import {isNullOrUndefined} from 'util';
 import {debounceTime, filter} from 'rxjs/operators';
+import {FormControl} from '@angular/forms';
+import {CollectionValidators} from '../../../core/validators/collection.validators';
+import {PostItCollectionNameComponent} from '../collection-name/post-it-collection-name.component';
 
 @Component({
-  selector: 'post-it-collection',
+  selector: '[post-it-collection]',
```

Comment:
What is the reason behind changing this? I think this change just complicates things without any added value.